### PR TITLE
changed log level of exceptions happening in tasks from `WARNING` to `ERROR`

### PR DESCRIPTION
**Pull request by syncblaze**
### Summary
changed log level of exceptions happening in tasks from `WARNING` to `ERROR`

### Checklist

- [ ] I have run `nox` and all the pipelines have passed.
- [x] I have created a changelog fragment to describe this change

### CODE REVIEW

**Comment by syncblaze** [comment]
i would assume a fragment is not needed so i just ignored that, if it needs one tell me i'll add it

File: lightbulb/tasks.py [hunk 1]
```diff
@@ -315,12 +315,12 @@ async def _loop(self) -> None:
                     n_failures += 1
 
                     if self._max_failures <= 0:
-                        LOGGER.warning(
+                        LOGGER.error(
                             "Execution of tast %r failed", self._func.__name__, exc_info=(type(e), e, e.__traceback__)
```
Comment (null-domain) [inline lightbulb/tasks.py:319]: Spelling mistake spotted! - `tast` instead of `task`. Could you fix that?

**Review by null-domain — COMMENTED** [review]
LGTM, and I agree with what you said on Discord -
> wouldnt it make much more sense to change the logging level of exceptions happening inside of tasks to error instead of warning? when an exception happens while loading extensions, the logging level is ERROR, when an exception happens during the execution of an command the level is ERROR but inside a task it logs as warning

Definitely seems like an inconsistency to me!

Personally I would say to include the fragment anyway - while it's such a small tweak that changes nothing functionally, it's still good to have it documented in the changelog.

**Comment by syncblaze** [comment]
will add the fragment in a few minutes

**Comment by syncblaze** [comment]
omg i am so stupid `bufix` 😭

it was 4 am dont judge me please 💀

**Pull request merged**